modules/pages/client/components/Home.component.js

```diff
@@ -0,0 +1,461 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { userType } from '@/modules/users/client/users.prop-types';
+import ManifestoText from './ManifestoText.component.js';
+import Tooltip from '@/modules/core/client/components/Tooltip.js';
+import BoardCredits from '@/modules/core/client/components/BoardCredits.js';
+import { Trans, useTranslation } from 'react-i18next';
+import classnames from 'classnames';
+import Board from '@/modules/core/client/components/Board.js';
+import { getRouteParams } from '@/modules/core/client/services/angular-compat';
+import getTribeBackgroundStyle from '@/modules/tribes/client/components/helpers/getTribeBackgroundStyle';
+import * as tribesAPI from '@/modules/tribes/client/api/tribes.api';
+
+const api = {
+  tribes: tribesAPI,
+};
+
+export default function Home({ user, isNativeMobileApp, photoCredits }) {
+  const { t } = useTranslation('pages');
+  const { tribe: tribeRoute } = getRouteParams();
+
+  // TODO get header height instead of magic number 56
+  // const headerHeight = angular.element('#tr-header').height() || 0; // code of the original angular controller
+  const headerHeight = 56;
+
+  const boardHeight =
+    window.innerWidth <= 480 && window.innerHeight < 700
+      ? 400
+      : window.innerHeight - headerHeight + 14;
+
+  const boards =
+    tribeRoute &&
+    ['hitchhikers', 'dumpster-divers', 'punks'].indexOf(tribeRoute) > -1
+      ? // Photos for these 3 tribes
+        [
+          'rainbowpeople',
+          'hitchroad',
+          'desertgirl',
+          'hitchgirl1',
+          'hitchgirl2',
+          'hitchtruck',
+        ]
+      : [
+          'woman-bridge',
+          'rainbowpeople',
+          'hitchroad',
+          'hitchgirl1',
+          'wavewatching',
+          'sahara-backpacker',
+          'hitchtruck',
+        ];
+
+  const [tribes, setTribes] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      const tribes = await api.tribes.read({ limit: 3 });
+      const tribeIsLoaded = tribes.some(t => t.slug === tribeRoute);
+
+      if (tribeRoute && !tribeIsLoaded) {
+        const extraTribe = await api.tribes.get(tribeRoute);
+
+        if (extraTribe && extraTribe._id) {
+          tribes.unshift(extraTribe);
+        }
+      }
+      setTribes(tribes);
```

Comment:
I implemented the behaviour of my understanding of the angular home controller, but it behaves differently compared to production. Is that a bug in production or should this be different? @simison